src/artwork.c

```diff
@@ -1515,7 +1589,7 @@ source_item_stream_get(struct artwork_ctx *ctx)
 {
   struct db_queue_item *queue_item;
   char *url;
-  char *ext;
+  //char *ext;
```

Comment:
What's up with the commenting out instead of removing? We do have version control.